### PR TITLE
fix: HS-166: New Elements fle without PreMountLoader

### PR DESCRIPTION
**Pull request by ArushKapoorJuspay**
**Problem Description -** New Elements File without the PreMountLoader

### CODE REVIEW

**Review by prafulkoppalkar — DISMISSED** [review]

File: src/orca-loader/Elements.res [hunk 1]
```diff
@@ -18,6 +18,7 @@ type trustPayFunctions = {
 let make = (
   options,
   setIframeRef,
+  ~clientSecretId,
```
Comment (vsrivatsa-juspay) [inline src/orca-loader/Elements.res:21]:
```suggestion
  ~clientSecret,
```

File: src/orca-loader/Elements.res [hunk 2]
```diff
@@ -44,6 +44,21 @@ let make = (
...
+    let switchToCustomPod = GlobalVars.isInteg
+      ? options
+        ->Js.Json.decodeObject
+        ->Belt.Option.flatMap(x => x->Js.Dict.get("switchToCustomPodABP"))
+        ->Belt.Option.flatMap(Js.Json.decodeBoolean)
+        ->Belt.Option.getWithDefault(false)
+      : false
+
     let paymentMethodListPromise = PaymentHelpers.usePaymentMethodList(
       ~clientSecret=clientSecretId,
```
Comment (vsrivatsa-juspay) [inline src/orca-loader/Elements.res:63]:
```suggestion
      ~clientSecret,
```

File: src/orca-loader/Hyper.res [hunk 3]
```diff
@@ -336,6 +336,7 @@ let make = (publishableKey, options: option<Js.Json.t>, analyticsInfo: option<Js
           setIframeRef,
           ~sdkSessionId=sessionID,
           ~publishableKey,
+          ~clientSecretId,
```
Comment (vsrivatsa-juspay) [inline src/orca-loader/Hyper.res:339]:
```suggestion
          ~clientSecret,
```

File: src/orca-loader/Elements.res [hunk 4]
```diff
@@ -44,6 +44,21 @@ let make = (
       ->Belt.Option.getWithDefault([])
       ->Js.Json.array
 
+    let blockConfirm = GlobalVars.isInteg
```
Comment (vsrivatsa-juspay) [inline src/orca-loader/Elements.res:47]: let blockConfirm = GlobalVars.isInteg
      && options
        ->Js.Json.decodeObject
        ->Belt.Option.flatMap(x => x->Js.Dict.get("orcaBlockConfirmABP"))
        ->Belt.Option.flatMap(Js.Json.decodeBoolean)
        ->Belt.Option.getWithDefault(false)

File: src/orca-loader/Elements.res [hunk 5]
```diff
@@ -44,6 +44,21 @@ let make = (
...
+      ? options
+        ->Js.Json.decodeObject
+        ->Belt.Option.flatMap(x => x->Js.Dict.get("orcaBlockConfirmABP"))
+        ->Belt.Option.flatMap(Js.Json.decodeBoolean)
+        ->Belt.Option.getWithDefault(false)
+      : false
+    let switchToCustomPod = GlobalVars.isInteg
```
Comment (vsrivatsa-juspay) [inline src/orca-loader/Elements.res:54]: let switchToCustomPod = GlobalVars.isInteg && options
        ->Js.Json.decodeObject
        ->Belt.Option.flatMap(x => x->Js.Dict.get("switchToCustomPodABP"))
        ->Belt.Option.flatMap(Js.Json.decodeBoolean)
        ->Belt.Option.getWithDefault(false)

File: src/orca-loader/Elements.res [hunk 6]
```diff
@@ -223,6 +238,9 @@ let make = (
             ("publishableKey", publishableKey->Js.Json.string),
             ("sdkSessionId", sdkSessionId->Js.Json.string),
             ("sdkHandleConfirmPayment", sdkHandleConfirmPayment->Js.Json.boolean),
+            ("AOrcaBBlockPConfirm", blockConfirm->Js.Json.boolean),
```
Comment (vsrivatsa-juspay) [inline src/orca-loader/Elements.res:241]: simplify key name

File: src/orca-loader/Elements.res [hunk 7]
```diff
@@ -223,6 +238,9 @@ let make = (
...
+            ("switchToCustomPodABP", switchToCustomPod->Js.Json.boolean),
```
Comment (vsrivatsa-juspay) [inline src/orca-loader/Elements.res:242]: simplify key name

**Review by vsrivatsa-juspay — APPROVED** [review]

**Review by prafulkoppalkar — APPROVED** [review]

**Pull request merged**